portfolio/src/components/Projects.tsx

```diff
@@ -1,71 +0,0 @@
-import React, { Component } from 'react';
-import Container from 'react-bootstrap/Container';
-import Row from 'react-bootstrap/Row';
-import Col from 'react-bootstrap/Col';
-import Button from 'react-bootstrap/Button';
-import projects from '../data/projects.json';
-
-
-type Props = {
-
-}
-type State = {
-    projects: {
-        name: string,
-        stack: string[],
-        headline: string,
-        description: string,
-        img: string,
-        source: string,
-        demo: string
-    }[]
-}
-export default class Projects extends Component<Props, State>{
-    constructor(props: Props) {
-        super(props);
-    }
-    componentWillMount() {
-        this.setState({
-            projects: []
-        });
-    }
-    componentDidMount() {
-        console.log(projects);
-        this.setState({
-            projects: projects
-        });
-    }
-    render() {
-        return (
-            <Container fluid={true} className="projects-container">
-                <h2 className="header header-main">Projects</h2>
-                {
-                    this.state.projects.map((p, i) => {
-                        return (
-                            <Row key={i}>
-                                <Col md={5} className="project-img">
-                                    <img className="img" src={p.img} />
-                                </Col>
-                                <Col md={7} className="project-content">
-                                    <h1 className="name">{p.name}</h1>
-                                    <h3 className="headline">{p.headline}</h3>
-                                    <h3 className="stack">
-                                        {
-                                            p.stack.map((s, i) => {
-                                                return <span>{(i ? ', ' : '') + s}</span>
-                                            })
```

Comment:
unclear what this is doing at first glance. Code readability is better than being concise.